deployments/forgeops/deploy.sh

```diff
@@ -83,8 +83,8 @@ echo "====================================================="
 echo "Building AM and IDM"
 echo "====================================================="
 cd $FORGEOPS_DIR/bin
-./forgeops build am --config-profile $CONFIG_PROFILE
-./forgeops build idm --config-profile $CONFIG_PROFILE
+./forgeops build am --config-profile $CONFIG_PROFILE --default-repo $CONTAINER_REGISTRY
+./forgeops build idm --config-profile $CONFIG_PROFILE --default-repo $CONTAINER_REGISTRY
```

Comment:
Can you please add `CONTAINER_REGISTRY` to the:

- check on line 25 of this file
- env print out on line 47 of this file
- env list on line 19 of `deployments/forgeops/README.md`